howtos/diversification.rst

```diff
@@ -0,0 +1,37 @@
+Diversification of Search Results
+=================================
+
+.. contents:: Table of contents
+
+Introduction
+------------
+
+Xapian allows for diversification of documents which are stored in the form of an MSet.
+This feature is a well-known technique in information retrieval used to increase
+user satisfaction, especially for ambiguous queries.
+
+Xapian currently has an implementation of an *implicit* method (using documents as features,
+as opposed to using query based features such as query logs) adapted from the C :sup:`2` - GLS method mentioned in Scalable and Efficient Web Search Results Diversification, Naini et al. 2016. This saves the cost of not having to provide external features such as query
```

Comment:
I'd probably say "this saves the cost of having to provide", rather than "not having to provide".

Reply:
Yes, that reads better - pushed as a fixup commit.